packages/gestalt/src/Link.js

```diff
@@ -29,6 +30,11 @@ type Props = {|
    * Use `accessibilitySelected` and `role` when using it as a tab. See the [Accessibility guidelines](https://gestalt.pinterest.systems/link#Accessible-Tab-Link) for more information.
    */
   accessibilitySelected?: boolean,
+  /**
+   * Auxiliary data passed to custom logic from Gestalt providers. See [OnInteractionProvider](https://gestalt.pinterest.systems/oninteractionprovider) to learn more.
+   */
+  // $FlowFixMe[unclear-type]
+  aux?: {| onInteractionData?: { [string]: any } |},
```

Comment:
This name does seem a bit specific to Pinboard. What would a more generic description of this functionality be?